forge-gui/res/cardsfolder/upcoming/trash_the_town.txt

```diff
@@ -0,0 +1,13 @@
+Name:Trash the Town
+ManaCost:G
+Types:Instant
+K:Spree
+A:SP$ Charm | Choices$ DBCounters,DBTrample,DBDrawTwo | MinCharmNum$ 1 | CharmNum$ 3 | Spree$ True
+SVar:DBCounters:DB$ PutCounter | SpreeCost$ 2 | ValidTgts$ Creature | TgtPrompt$ Select target creature | CounterType$ P1P1 | CounterNum$ 2 | SpellDescription$ Put two +1/+1 counters on target creature.
+SVar:DBTrample:DB$ Pump | SpreeCost$ 1 | ValidTgts$ Creature | TgtPrompt$ Select target creature | KW$ Trample | SpellDescription$ Target creature gains trample until end of turn.
+SVar:DBDrawTwo:DB$ Pump | SpreeCost$ 1 | ValidTgts$ Creature | TgtPrompt$ Select target creature | SubAbility$ DBAnimate | SpellDescription$ Until end of turn, target creature gains "Whenever this creature deals combat damage to a player, draw two cards."
+SVar:DBAnimate:DB$ Animate | Triggers$ TrigCDPlayer | Defined$ ParentTarget | StackDescription$ None
```

Comment:
why not target directly with Animate?

Reply:
That does work 👍 
Here I referenced Feign Death, mostly to make sure I wasn't skipping steps. I take it then Feign Death's script and similar ones could use a cleanup?

Reply:
sure